docs/develop/getting-started/quick_start.md

```diff
@@ -25,17 +25,21 @@ The easiest way to install WasmEdge is to run the following command. You should
 curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | sudo bash -s -- -p /usr/local
 ```
 
-> If you do not have root or `sudo` rights, use the following line to install WasmEdge in your `$HOME/.wasmedge` directory: `curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash`
+> If you do not have root or `sudo` rights, use the following line to install WasmEdge in your `$HOME/.wasmedge` directory:
+
+```bash
+ `curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash`
+```
 
 
-## Run a standalone Wasm app
+## How to run a standalone Wasm app
 
 The Hello world example is a standalone Rust application that can be executed by the [WasmEdge CLI](../build-and-run/cli). Its source code and build instructions are available [here](https://github.com/second-state/rust-examples/tree/main/hello).
 
-[Download the hello.wasm file here](/files/hello.wasm)
+Download the hello.wasm file [here](https://wasmedge.org/docs/files/hello.wasm), or run the following command:
 
 ```bash
-wget https://wasmedge.org/books/app-dev-guide/en/files/hello.wasm
```

Comment:
I think we need to use this url. The documentation and the sample file it refers to should ideally be in the same repo.

Reply:
Agreed, it makes a lot of sense in doing that way and can avoid tempering files too by building it in actions and storing it artifact. I was wondering which will be a better way for building all examples using Cargo Workspace or building it by triggering each example to build?

I will create a PR in

Reply:
> I think we need to use this url. The documentation and the sample file it refers to should ideally be in the same repo.

Okay, noted.
Thanks for reviewing!